packages/components/src/interfaces/native/Prose.ts

```diff
@@ -39,12 +43,15 @@ const generateProseSchema = (id?: string) => {
     {
       ...(id && { $id: id }),
       ...BASE_PROSE_META,
+      format,
```

Comment:
added a discriminant property here so that we can switch to render the editor in `studio`